integreat_cms/cms/models/regions/region.py

```diff
@@ -370,6 +375,24 @@ class Region(AbstractBaseModel):
         verbose_name=_("used DeepL budget"),
     )
 
+    machine_translate_pages = models.PositiveIntegerField(
+        choices=machine_translation_permissions.CHOICES,
+        default=machine_translation_permissions.EVERYONE,
+        verbose_name=_("Pages"),
+    )
+
+    machine_translate_events = models.PositiveIntegerField(
+        choices=machine_translation_permissions.CHOICES,
+        default=machine_translation_permissions.EVERYONE,
+        verbose_name=_("Events"),
+    )
+
+    machine_translate_pois = models.PositiveIntegerField(
+        choices=machine_translation_permissions.CHOICES,
+        default=machine_translation_permissions.EVERYONE,
+        verbose_name=_("POIs"),
```

Comment:
```suggestion
        verbose_name=_("Locations"),
```
`POIs` is only our internal model name for historic reasons, in the UI we use the term `Location`.